storage/tests/unit/test_blob.py

```diff
@@ -312,6 +313,32 @@ def test_generate_signed_url_w_method_arg(self):
         }
         self.assertEqual(SIGNER._signed, [(EXPECTED_ARGS, EXPECTED_KWARGS)])
 
+    @mock.patch('google.cloud.storage._signing.get_signed_query_params',
+                return_value={
+                    'GoogleAccessId': 'service-account-name',
+                    'Expires': 12345,
+                    'Signature': 'signed-data',
+                })
+    def test_generate_resumable_signed_url(self, mock_get_signed_query_params):
+        """
+        Verify correct behavior of resumable upload URL generation
+        """
+        from google.cloud.storage._signing import generate_signed_url
```

Comment:
nit: empty newline between imports and code.

Reply:
Done.